### PR TITLE
Dev

### LINKED ISSUE #1: [Feature Request] Create cross-compiled debian package of orocos-kdl via docker
### ⚡ Feature Request

See [comment](https://github.com/107-systems/l3xz/pull/9#issuecomment-1152020204).

### PR DESCRIPTION
**Pull request by wuehr1999**
Fixes #1

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Create package
+##
+mkdir DEBIAN
+cd DEBIAN
+echo "Package: kdl-107-$1
+Version: 0.3.13
+Architecture: all
+Maintainer: Jonas Wühr 
+Depends: libeigen3-dev 
+Homepage: https://github.com/shinokada/tera
```
Comment (aentinger) [inline build.sh]: Not sure what belongs here, but this does not seem right =)

**Review by aentinger — APPROVED** [review]
Great work! Thank you very much @wuehr1999 :rocket: 

I was wondering if you could change the output name of the debian package though: `liborocos-kdl-dev-${VERSION or GITHASH}-${DATE]-${ARCH}.deb`?

Also I'd like to invite @per1234 to give this PR a review :pray: and suggestions for possible improvements.

**Comment by aentinger** [comment]
Note @wuehr1999 : Please use "Squash & Merge" when merging this PR. Thank you :bow:

File: build.sh (hunk 1 above)
Comment (wuehr1999) [reply build.sh]: Hi @aentinger The Homepage is simply wrong, as I took a random debian package as a template ;-) 
Architecture: all is a little bit hacky, so I will improve the control file

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+**.deb
+orocos_knematics_dynamics
```
Comment (aentinger) [inline .gitignore]:
```suggestion
orocos_kinematics_dynamics
```
I venture to suggest that there's a typo here :wink:

**Pull request merged**